docs/tools/terraform/reference/cookbook/kafka-flink-integration-recipe.rst

```diff
@@ -0,0 +1,112 @@
+
+Describe the setup
+------------------
+
+The following Terraform script stands up both Apache Kafka and Apache Flink services, creates the service integration and  related resources. The parameters and configurations will vary for your case and you can find
+links to the advanced parameters as a reference at the end of this document.
+
+
+``services.tf`` file:
+
+.. code:: bash
+
+  resource "aiven_flink" "flink" {
+   project      = data.aiven_project.dev-advocates.project
+   cloud_name   = "google-europe-west1"
+   plan         = "business-8"
+   service_name = "demo-flink"
+  }
+
+  resource "aiven_kafka" "kafka" {
+   project      = data.aiven_project.dev-advocates.project
+   cloud_name   = "google-europe-west1"
+   plan         = "business-8"
+   service_name = "demo-kafka"
+  }
+
+  resource "aiven_service_integration" "flink_to_kafka" {
+   project                  = data.aiven_project.dev-advocates.project
+   integration_type         = "flink"
+   destination_service_name = aiven_flink.flink.service_name
+   source_service_name      = aiven_kafka.kafka.service_name
+  }
+
+  resource "aiven_kafka_topic" "source" {
+   project      = data.aiven_project.dev-advocates.project
+   service_name = aiven_kafka.kafka.service_name
+   partitions   = 2
+   replication  = 3
+   topic_name   = "source_topic"
+  }
+
+  resource "aiven_kafka_topic" "sink" {
+   project      = data.aiven_project.dev-advocates.project
+   service_name = aiven_kafka.kafka.service_name
+   partitions   = 2
+   replication  = 3
+   topic_name   = "sink_topic"
+  }
+
+  resource "aiven_flink_table" "source" {
+   project        = data.aiven_project.dev-advocates.project
+   service_name   = aiven_flink.flink.service_name
+   integration_id = aiven_service_integration.flink_to_kafka.integration_id
+   table_name     = "source_table"
+   kafka_topic    = aiven_kafka_topic.source.topic_name
+   schema_sql     = <<EOF
+    `cpu` INT,
+    `node` INT,
+    `occurred_at` TIMESTAMP(3) METADATA FROM 'timestamp',
+    WATERMARK FOR `occurred_at` AS `occurred_at` - INTERVAL '5' SECOND
+   EOF
+  }
+
+  resource "aiven_flink_table" "sink" {
+   project        = data.aiven_project.dev-advocates.project
+   service_name   = aiven_flink.flink.service_name
+   integration_id = aiven_service_integration.flink_to_kafka.integration_id
+   table_name     = "sink_table"
+   kafka_topic    = aiven_kafka_topic.sink.topic_name
+   schema_sql     = <<EOF
+     `cpu` INT,
+     `node` INT,
+     `occurred_at` TIMESTAMP(3)
+   EOF
+  }
+
+  resource "aiven_flink_job" "flink_job" {
+   project      = data.aiven_project.dev-advocates.project
+   service_name = aiven_flink.flink.service_name
+   job_name     = "my_job"
+   table_ids = [
+     aiven_flink_table.source.table_id,
+     aiven_flink_table.sink.table_id
+   ]
+   statement = <<EOF
+     INSERT INTO ${aiven_flink_table.sink.table_name}
+     SELECT * FROM ${aiven_flink_table.source.table_name}
+     WHERE `cpu` > 70
```

Comment:
Again, this is great! Let's mention it in the text and explain what the story is for the sample job, and that people can change it